src/anomalib/models/efficientad/config.yaml

```diff
@@ -0,0 +1,103 @@
+dataset:
+  name: mvtec
+  format: mvtec
+  path: ./datasets/MVTec
+  category: bottle
+  task: segmentation
+  train_batch_size: 1
+  eval_batch_size: 1
+  num_workers: 0
+  image_size: 256 # dimensions to which images are resized (mandatory)
+  center_crop: null # dimensions to which images are center-cropped after resizing (optional)
+  normalization: imagenet # data distribution to which the images will be normalized: [none, imagenet]
+  transform_config:
+    train: null
+    eval: null
+  test_split_mode: from_dir # options: [from_dir, synthetic]
+  test_split_ratio: 0.2 # fraction of train images held out testing (usage depends on test_split_mode)
+  val_split_mode: same_as_test # options: [same_as_test, from_test, synthetic]
+  val_split_ratio: 0.5 # fraction of train/test images held out for validation (usage depends on val_split_mode)
+
+model:
+  name: efficientad
+  pre_trained_dir: src/anomalib/models/efficientad/pre_trained/
+  teacher_file_name: teacher_medium_formatted_nelson1425.pth
+  teacher_out_channels: 384
+  model_size: M
+  lr: 0.0001
+  weight_decay: 0.00001
+  padding: false
+  # generic params
+  normalization_method: min_max # options: [null, min_max, cdf]
+
+metrics:
+  image:
+    - F1Score
+    - AUROC
+  pixel:
+    - F1Score
+    - AUROC
+  threshold:
+    method: adaptive #options: [adaptive, manual]
+    manual_image: null
+    manual_pixel: null
+
+visualization:
+  show_images: False # show images on the screen
+  save_images: True # save images to the file system
+  log_images: False # log images to the available loggers (if any)
+  image_save_path: null # path to which images will be saved
+  mode: full # options: ["full", "simple"]
+
+project:
+  seed: 42
+  path: ./results
+
+logging:
+  logger: [] # options: [comet, tensorboard, wandb, csv] or combinations.
+  log_graph: false # Logs the model graph to respective logger.
+
+optimization:
+  export_mode: null # options: torch, onnx, openvino
+# PL Trainer Args. Don't add extra parameter here.
+trainer:
+  enable_checkpointing: true
+  default_root_dir: null
+  gradient_clip_val: 0
+  gradient_clip_algorithm: norm
+  num_nodes: 1
+  devices: 1
+  enable_progress_bar: true
+  overfit_batches: 0.0
+  track_grad_norm: -1
+  check_val_every_n_epoch: 1 # Don't validate before extracting features.
```

Comment:
Training of EfficientAD is steps instead of epochs, so validate every 1000 steps?

```
  check_val_every_n_epoch: null
  val_check_interval: 1000 # Validate every 1000 steps
```